daemon/graphdriver/driver.go

```diff
@@ -27,6 +27,8 @@ const (
 	FsMagicJffs2Fs     = FsMagic(0x000072b6)
 	FsMagicZfs         = FsMagic(0x2fc12fc1)
 	FsMagicXfs         = FsMagic(0x58465342)
+	FsMagicJfs         = FsMagic(0x3153464a)
+	FsMagicF2fs        = FsMagic(0xF2F52010)
```

Comment:
Just noticed the magic bits seem to be in random order, or is there some logic behind this?  (Not introduced by this PR, so feel free to ignore)